### DISCUSSION TITLE
Remote: Is it possible to use JavaScript files stored on the server?

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/Slinger#587] [General]
I have multiple remotes HTML files that share some functionality implemented with JavaScript, so I've tried to extract that functionality into a common script.

I've placed this script in a directory in the root and referenced it in my remote html with: 
` <script src="/js/slingerplayer.js"></script>`
 
 Unfortunately it doesn't work, so  I had to put the JS code back into the HTML file.
 
 Am I doing something wrong, or is it somewhere a setting I need to change to enable using JavaScript files?

**Comment by GerryDazoo** [comment]
Just saying it doesn't work isn't very helpful.  All the JS execution is done in your browser so that's where to start looking. There shouldn't be a reason why calling this script won't work.

**Reply by bradut** [reply]
**It works !**
You are right, a bit more context would have helped.
I inspected the code again before replying, and noticed that by mistake, in code I've commented out I've commented a closing </script>

Sorry for this and thank you for your quick reply.
